doc/api/events.md

```diff
@@ -935,13 +935,415 @@ if the `EventEmitter` emits `'error'`. It removes all listeners when
 exiting the loop. The `value` returned by each iteration is an array
 composed of the emitted event arguments.
 
+## `EventTarget` and `Event` API
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+The global `EventTarget` and `Event` objects are a Node.js-specific adaptation
+of the [`EventTarget` Web API][].
+
+```js
+const eventTarget = new EventTarget();
+const event = new Event('foo');
+
+eventTarget.addEventListener('foo', (event) => {
+  console.log(`The ${event.type} event happened`);
+});
+```
+
+The `EventTarget` class is similar in function to `EventEmitter` by evolved
+separately as part of browser DOM APIs. The Node.js implementation of
+`EventTarget` is an adaptation of the DOM API with a number of Node.js-specific
+extensions and behaviors intended to allow it to work more naturally within
+the Node.js environment.
+
+### Node.js `EventTarget` vs. DOM `EventTarget`
+
+There are four key differences between the Node.js `EventTarget` and the
+[`EventTarget` Web API][]:
+
+1. There is no concept of a hierarchical DOM and event propagation in Node.js.
+   That is, an event dispatched to an `EventTarget` does not propagate through
+   a hierarchy of nested target objects that may each have their own set of
+   handlers for the event.
+2. In the Node.js `EventTarget`, any object with a `type` property whose value
+   is a string may be dispatched as an `event`.
+3. In the Node.js `EventTarget`, if an event listener is an async function
```

Comment:
Are we allowed to do this?

Reply:
Yes, we're being explicit about how we differ from the browser-side API and I'm accounting for things that users will definitely end up doing anyway.